main/src/library/Zip.flix

```diff
@@ -0,0 +1,75 @@
+namespace Zip {
```

Comment:
How about something like `Core/Io/Zip`?

Reply:
Should I move this file into the appropriate namespace-folder structure too?

Reply:
Yes

Reply:
So just to be clear, since you wrote what I'd consider the opposite for `InputStream`, should I move the two files to `main/src/library/Core/Io/*.flix`?

Reply:
Yes